client/store/assessment/stateType.ts

```diff
@@ -0,0 +1,10 @@
+import { Assessment } from '@core/meta/assessment'
+import { Country, Region } from '@core/country'
+import { RegionGroup } from '@core/meta/regionGroup'
+
+export interface AssessmentState {
+  assessment?: Assessment
+  countries?: Array<Country>
+  regions?: Array<Region>
+  regionGroups?: Array<RegionGroup>
```

Comment:
Should we use partial?

Reply:
what do you think about this structure:

```
export interface AssessmentState {
   assessment?: Assessment
   countries?: Array<Country>
   regionGroups?: Record<string, RegionGroup>
```
and RegionGroup has also an array of regions as property.
this way the /api/init could return such structure.

then your useRegionGroups hook will return an array of region groups sorted by order